test/utils/image/manifest.go

```diff
@@ -47,42 +48,66 @@ func (i *ImageConfig) SetVersion(version string) {
-	Nettest            = ImageConfig{e2eRegistry, "nettest", "1.0"}
-	NginxSlim          = ImageConfig{gcRegistry, "nginx-slim", "0.20"}
-	NginxSlimNew       = ImageConfig{gcRegistry, "nginx-slim", "0.21"}
-	Nonewprivs         = ImageConfig{e2eRegistry, "nonewprivs", "1.0"}
-	NoSnatTest         = ImageConfig{e2eRegistry, "no-snat-test", "1.0"}
-	NoSnatTestProxy    = ImageConfig{e2eRegistry, "no-snat-test-proxy", "1.0"}
-	NWayHTTP           = ImageConfig{e2eRegistry, "n-way-http", "1.0"}
-	Pause              = ImageConfig{gcRegistry, "pause", "3.0"}
-	Porter             = ImageConfig{e2eRegistry, "porter", "1.0"}
-	PortForwardTester  = ImageConfig{e2eRegistry, "port-forward-tester", "1.0"}
-	Redis              = ImageConfig{e2eRegistry, "redis", "1.0"}
-	ResourceConsumer   = ImageConfig{e2eRegistry, "resource-consumer", "1.3"}
-	ResourceController = ImageConfig{e2eRegistry, "resource-consumer/controller", "1.0"}
-	ServeHostname      = ImageConfig{e2eRegistry, "serve-hostname", "1.0"}
-	TestWebserver      = ImageConfig{e2eRegistry, "test-webserver", "1.0"}
+	AdmissionWebhook         = ImageConfig{e2eRegistry, "k8s-sample-admission-webhook", "1.8v7", true}
+	APIServer                = ImageConfig{e2eRegistry, "k8s-aggregator-sample-apiserver", "1.7v2", true}
+	AppArmorLoader           = ImageConfig{gcRegistry, "apparmor-loader", "0.1", false}
+	BusyBox                  = ImageConfig{gcRegistry, "busybox", "1.24", false}
+	CheckMetadataConcealment = ImageConfig{gcRegistry, "check-metadata-concealment", "v0.0.2", false}
+	ClusterTester            = ImageConfig{e2eRegistry, "clusterapi-tester", "1.0", true}
+	CudaVectorAdd            = ImageConfig{e2eRegistry, "cuda-vector-add", "1.0", true}
+	Dnsutils                 = ImageConfig{e2eRegistry, "dnsutils", "1.0", true}
+	DNSMasq                  = ImageConfig{gcRegistry, "k8s-dns-dnsmasq", "1.14.5", true}
+	EchoServer               = ImageConfig{gcRegistry, "echoserver", "1.6", false}
+	EntrypointTester         = ImageConfig{e2eRegistry, "entrypoint-tester", "1.0", true}
+	E2ENet                   = ImageConfig{gcRegistry, "e2e-net", "1.0", true}
+	Fakegitserver            = ImageConfig{e2eRegistry, "fakegitserver", "1.0", true}
+	GBFrontend               = ImageConfig{sampleRegistry, "gb-frontend", "v5", true}
+	GBRedisSlave             = ImageConfig{sampleRegistry, "gb-redisslave", "v2", true}
+	Goproxy                  = ImageConfig{e2eRegistry, "goproxy", "1.0", true}
+	Hostexec                 = ImageConfig{e2eRegistry, "hostexec", "1.0", true}
+	Iperf                    = ImageConfig{e2eRegistry, "iperf", "1.0", true}
+	JessieDnsutils           = ImageConfig{e2eRegistry, "jessie-dnsutils", "1.0", true}
+	Kitten                   = ImageConfig{e2eRegistry, "kitten", "1.0", true}
+	Liveness                 = ImageConfig{e2eRegistry, "liveness", "1.0", true}
+	LogsGenerator            = ImageConfig{e2eRegistry, "logs-generator", "v0.1.0", true}
```

Comment:
I can't pull this:
```
$ docker pull gcr.io/kubernetes-e2e-test-images/logs-generator-amd64:v0.1.0
Trying to pull repository gcr.io/kubernetes-e2e-test-images/logs-generator-amd64 ...
manifest for gcr.io/kubernetes-e2e-test-images/logs-generator-amd64:v0.1.0 not found
```

I can pull this
```
docker pull gcr.io/kubernetes-e2e-test-images/logs-generator-amd64:1.0
Trying to pull repository gcr.io/kubernetes-e2e-test-images/logs-generator-amd64 ...
sha256:d0b043e375650c97c7023f800d20fed1d6cdbc07eb4af853fd6bc79f1334acbf: Pulling from gcr.io/kubernetes-e2e-test-images/logs-generator-amd64
3feb4c394fe4: Pull complete
df2c5969a9ef: Pull complete
Digest: sha256:d0b043e375650c97c7023f800d20fed1d6cdbc07eb4af853fd6bc79f1334acbf
Status: Downloaded newer image for gcr.io/kubernetes-e2e-test-images/logs-generator-amd64:1.0
```

Reply:
I'm not sure this image is used anywhere?

Reply:
Ah, it will be by logging_pod.go, which you switched to use this. I do think the tag needs to be fixed.

Reply:
Or you need to do gcRegistry without the arch (which does have v0.1.0)